README.md

```diff
@@ -1 +1,83 @@
-# git
\ No newline at end of file
+<!-- Headings -->
+# Sample .md file
+## This is a sub heading
+### Another sub heading 
+---
+
+<!-- italics-->
+_This text in italic_
+___
+
+<!--bold-->
+__This text in bold__
+___
+
+<!--strikethrough-->
+~~Strikethrough~~
+___
+
+<!--Horizontal Rule-->
+<!--- ___>
+
+<!--BlockQuote-->
+>This is a block quote
+___
+
+<!--Links-->
+[GitHub](https://github.com/KaranVyas "GitHub profile for Karan Vyas")
+___
+
+<!--Unordered List-->
+* Item 1
+* Item 2
+* Item 3
+___
+
+<!--Ordered List-->
+1. Item 1
+1. Item 2
+1. Item 3
+___
+
+<!--Inline Code Block-->
+`This is a inline code block`
+___
+
+<!--Images-->
+![Image](https://logos-download.com/wp-content/uploads/2020/06/Boston_University_Logo_text.png)
```

Comment:
Create a separate folder, store all images there, and try to fetch them from that folder.

Reply:
Uploaded files to Image folder in `LEARN-MD-FILES` branch